opendbc/car/hyundai/values.py

```diff
@@ -32,6 +32,13 @@ def __init__(self, CP):
       self.STEER_DELTA_UP = 2
       self.STEER_DELTA_DOWN = 3
 
+    elif CP.flags & HyundaiFlags.CAN_CANFD_BLENDED:
+      self.STEER_MAX = 270  # CAN FD max steer
+      self.STEER_DRIVER_ALLOWANCE = 50  # CAN driver torque signal scaling
+      self.STEER_THRESHOLD = 150  # CAN driver torque signal scaling
+      self.STEER_DELTA_UP = 2  # CAN FD rate limits
+      self.STEER_DELTA_DOWN = 3  # CAN FD rate limits
+
```

Comment:
we should make classes/constants for the duplicate values, ideally we just define it has CAN FD steering torque, but CAN driver torque